internal/olm/operator/internal/registry.go

```diff
@@ -190,9 +190,9 @@ func (rr *RegistryResources) DeletePackageManifestsRegistry(ctx context.Context,
 }
 
 // GetRegistryServiceAddr returns a Service's DNS name + port for a given
-// pkgName and namespace.
-func GetRegistryServiceAddr(pkgName, namespace string) string {
-	name := getRegistryServerName(pkgName)
+// resourceName and namespace.
+func GetRegistryServiceAddr(resourceName, namespace string) string {
+	name := getRegistryServerName(resourceName)
 	return fmt.Sprintf("%s.%s.svc.cluster.local:%d", name, namespace, registryGRPCPort)
 }
```

Comment:
```suggestion
// resourceName and namespace.
func (rp *GetRegistryServiceAddr) GetRegistryServiceAddr() string {
	return fmt.Sprintf("%s.%s.svc.cluster.local:%d",  getRegistryServerName(rp.Pod.podName), rp.Namespace, registryGRPCPort)
}
```

Reply:
I think you meant `rp *RegistryPod` as the receiver type. 
Cannot use it on this method as this method is called from the package manifests code as well.